ucte/ucte-network/src/main/java/com/powsybl/ucte/network/UcteValidation.java

```diff
@@ -33,8 +33,8 @@ public static void checkValidLineCharacteristics(UcteLine line) {
                 if (line.getResistance() < ZERO_EPS) {
                     LOGGER.error(lineId, "Real line resistance cannot be negative", line.getResistance() + " ohm");
                 }
-                if (Math.abs(line.getReactance()) > ZERO_EPS) {
-                    LOGGER.warn(lineId, "Busbar coupler reactance must be zero. For lines this value must be larger than 0.05 ohm",
+                if (Math.abs(line.getReactance()) < REACTANCE_EPS) {
+                    LOGGER.warn(lineId, "Real line reactance must be larger than 0.05 ohm",
                             line.getReactance() + " ohm");
                 }
```

Comment:
Does this rule exist? I didn't find it in the [Quality of Datasets and Calculations](https://eepublicdownloads.entsoe.eu/clean-documents/Publications/SOC/Continental_Europe/130125_Quality%20of%20Datasets%20and%20Calculations.pdf) (I agree that the previous code was faulty but I wonder if it just shouldn't be deleted?)